tests/phpunit/tests/xmlrpc/wp/getUsers.php

```diff
@@ -44,49 +44,55 @@ function test_capable_user() {
 
 	function test_invalid_role() {
 		$administrator_id = $this->make_user_by_role( 'administrator' );
-		if ( is_multisite() )
+		if ( is_multisite() ) {
 			grant_super_admin( $administrator_id );
+		}
 
-		$filter = array( 'role' => 'invalidrole' );
+		$filter  = array( 'role' => 'invalidrole' );
 		$results = $this->myxmlrpcserver->wp_getUsers( array( 1, 'administrator', 'administrator', $filter ) );
 		$this->assertIXRError( $results );
 		$this->assertEquals( 403, $results->code );
 	}
 
 	function test_role_filter() {
-		$author_id = $this->make_user_by_role( 'author' );
-		$editor_id = $this->make_user_by_role( 'editor' );
+		$author_id        = $this->make_user_by_role( 'author' );
+		$editor_id        = $this->make_user_by_role( 'editor' );
 		$administrator_id = $this->make_user_by_role( 'administrator' );
-		if ( is_multisite() )
+		if ( is_multisite() ) {
 			grant_super_admin( $administrator_id );
+		}
 
 		// test a single role ('editor')
-		$filter = array( 'role' => 'editor' );
+		$filter  = array( 'role' => 'editor' );
 		$results = $this->myxmlrpcserver->wp_getUsers( array( 1, 'administrator', 'administrator', $filter ) );
```

Comment:
Should items in this array start on new lines since it has more than one item?

Reply:
it's not an associative array